### ISSUE TITLE
[Feature]: Allow running checker via automation and store its findings in automation friendly format such as XML

### ISSUE THREAD

**Issue by swijenay** [IBMa/equal-access#2184]
### Project

accessibility-checker-engine

### Description

I'm a Software Development Manager for an IBM product named Cognos Analytics (CA). We are using the following  IBM Equal Access Accessibility Checker for identifying Accessibility violations in CA.  

https://addons.mozilla.org/en-CA/firefox/addon/accessibility-checker/
https://chromewebstore.google.com/detail/ibm-equal-access-accessib/lkcagbfjnkomcinoddgooolagloogehp

Our plan is to automate the Accessibility Testing of CA. As a part of this automation, we are hoping to open each web page of the application, run the checker against the opened page, and programatically analyze the findings of the checker to see if there are any new concerns. In order to do so, we'll need a way to run the checker via automation (which is based on Selenium) and then have its findings stored in an automation friendly format such as XML for automated analysis. 

Could you please make the above mentioned enhancement for us? Please let me know if this enhancement request (ticket) should be opened somewhere else. Feel free to triage this ticket if necessary. Thanks!

**Comment by tombrunet** [comment]
@swijenay We have several such tools. See the CI/CD section under the Develop section here:
https://www.ibm.com/able/toolkit/tools/#develop

**Comment by swijenay** [comment]
@tombrunet Thank you very much for the info. I think the Equal Access Checker for Java would do in my case: https://central.sonatype.com/artifact/com.ibm.able/accessibility-checker/overview. Do you have any usage info of this Maven module?

**Comment by philljenkins** [comment]
@swijenay there are some [boilerplates](https://github.com/IBMa/equal-access/tree/master/accessibility-checker/boilerplates) for the other Node packages, does that have the info you need for[ your Java](https://github.com/IBMa/equal-access/tree/master/java-accessibility-checker/boilerplates) case?

[Automated testing in your pipeline](https://github.com/IBMa/equal-access/wiki#automated-testing-in-your-pipeline)
